lib/Optimizer/CMakeLists.txt

```diff
@@ -2,13 +2,12 @@ add_library(Optimizer
               IROptimizer.cpp
               GraphOptimizer.cpp
               Lower.cpp
-              Quantization.cpp)
+              Quantization.cpp
+              TrainingPreparation.cpp)
 
 target_link_libraries(Optimizer
                       PRIVATE
                         Backend
-                        Converter
```

Comment:
Why did you remove dependencies here? This seems like an accidental change?

Reply:
Quantization library includes Converter

Reply:
Sure, but Optimizer directly depends on Converter itself as well. If Quantization somehow no longer depends on Converter in the future then this will break. Removing this just makes the dependency graph less accurate. Is there some downside to keeping Converter etc. here?